contracts/stabilizer/PSMRouter.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.4;
+
+import "./IPSMRouter.sol";
+import "./PegStabilityModule.sol";
+import "../Constants.sol";
+
+contract PSMRouter is IPSMRouter {
+
+    IPegStabilityModule public immutable psm;
```

Comment:
Do we want a way for governance to change this? I'm undecided

Reply:
No, we do not want to add governance to this contract. The router should be as simple as possible and if we want to change the router, we can always create a new one as the PSM does not have a reference to the router.